src/the-2021-scheduler.md

```diff
@@ -47,10 +47,10 @@ the backlog before you add more new cards. If you're sure you want to add more n
 even though you have a backlog, you can enable the "New cards ignore review limit" option
 in deck options.
 
-The limits of each deck control the number of cards that will be gathered from that deck
+In a deck tree, the limits of each deck will control the number of cards that gathered from that deck
 and its subdecks. Limits are applied from the deck you select, so if you select
-a child deck, its parents' limits will not apply. For example, with the
-following limits:
+a child deck, its parents' limits will not apply. For example, if the
+following are the limits:
```

Comment:
I prefer the original wording and don't think this needs to change, but if it changes, "are the limits" doesn't sound right to me. Perhaps this instead? 
```suggestion
a child deck, its parents' limits will not apply. For example, if the
following limits are set:
```